examples/pythonScripts/timeChipGaps.py

```diff
@@ -0,0 +1,58 @@
+#! /usr/bin/env python
+# A little script to time out the penalty for running chip gaps
+
+import timeit
+
+
+def runChips(useCamera=False):
+    import numpy as np
+    import lsst.sims.maf.slicers as slicers
+    import lsst.sims.maf.metrics as metrics
+    import lsst.sims.maf.metricBundles as metricBundles
+    import lsst.sims.maf.db as db
+    from lsst.sims.maf.plots import PlotHandler
+    import matplotlib.pylab as plt
+    import healpy as hp
+
+
+    print 'Camera setting = ', useCamera
+
+    dbAddress = 'sqlite:///enigma_1189_sqlite.db'
```

Comment:
dbAddress -> "database" (to be consistent with new db API) and "sqlite:///enigma_1189_sqlite.db" -> "enigma_1189_sqlite.db"